packages/app-mobile/root.tsx

```
@@ -103,14 +103,25 @@ import SamlShared from '@joplin/lib/components/shared/SamlShared';
 import buildStartupTasks from './utils/buildStartupTasks';
-import { SafeAreaProvider } from 'react-native-safe-area-context';
+import { SafeAreaProvider, useSafeAreaInsets } from 'react-native-safe-area-context';
 import appReducer from './utils/appReducer';
 import SyncWizard from './components/SyncWizard/SyncWizard';
 import Synchronizer from '@joplin/lib/Synchronizer';
 
 const logger = Logger.create('root');
 const perfLogger = PerformanceLogger.create();
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any -- DropdownAlert is untyped
+const SafeDropdownAlert = ({ alert }: { alert: (func: any)=> void }) => {
```

ESLint

[error] 'func' is defined but never used. Allowed unused args must match /^_/u.

(no-unused-vars)